fixes 1 13 (#472)
amplitude/amplitude-docs: content/collections/data/en/csv-import-export.md

4. Once you’re sure the imported tracking plan updates are correct, you can [merge the changes back into the main branch](/docs/data/data-planning-workflow).
5. If you want to export user properties from your tracking plan, click the export icon instead.
6. In the export modal that appears, choose whether to download your tracking plan’s schema or a .CSV file template. Then, click Download.

**![](/docs/output/img/data/22418912795163.png)**

The next section defines the user properties .CSV import file structure and the [additional columns seen in the user properties export](#user-properties-file-schema).

## User properties file schema

The list below highlights the required schema for the user properties .CSV file import. There are some things to note: 

* The file's header must contain all listed fields.
* The fields can be in any order, but their names and values must be exact in spelling and capitalization to ensure a successful import process.
* Some fields require a value as indicated in the following list.

**Action**
* Definition: Action to perform on a particular user property
* Values:
    * `Ignore`: No changes will be made to the user property.

>     * `Remove`: User property will be removed from the tracking plan. The remove value doesn't delete a user property, nor does it stop its ingestion.

Author: ```suggestion
    * `Remove`:  The user property is removed from the tracking plan. The removed value doesn't delete a user property, nor does it stop its ingestion.
```
